src/panels/config/automation/condition/types/ha-automation-condition-time.ts

```diff
@@ -160,7 +160,7 @@ export class HaTimeCondition extends LitElement implements ConditionElement {
           >
             <ha-switch
               .day=${day}
-              .checked=${weekday?.includes(day)}
+              .checked=${!weekday || weekday?.includes(day)}
```

Comment:
```suggestion
              .checked=${!weekday || weekday.includes(day)}
```

Reply:
weekday can btw also be a string, so that should also be checked, and the type adjusted

Reply:
done

Reply:
> weekday can btw also be a string, so that should also be checked, and the type adjusted

I've added string[] as a type: https://github.com/home-assistant/frontend/pull/6848/files#diff-be629ff01fe2971d4cb277c22d528d74R173